cura/CuraApplication.py

```diff
@@ -373,9 +373,17 @@ def __addExpectedResourceDirsAndSearchPaths(self):
             Resources.addExpectedDirNameInData(dir_name)
 
         app_root = os.path.abspath(os.path.join(os.path.dirname(sys.executable)))
-        Resources.addSecureSearchPath(os.path.join(app_root, "share", "cura", "resources"))
 
-        Resources.addSecureSearchPath(os.path.join(self._app_install_dir, "share", "cura", "resources"))
+        if platform.system() == "Darwin":
+            Resources.addSecureSearchPath(os.path.join(app_root, "Resources", "share", "cura", "resources"))
+            Resources.addSecureSearchPath(os.path.join(app_root, "..", "Resources", "share", "cura", "resources"))
+            Resources.addSecureSearchPath(
+                os.path.join(self._app_install_dir, "..", "Resources", "share", "cura", "resources"))
+        else:
+            Resources.addSecureSearchPath(os.path.join(app_root, "share", "cura", "resources"))
+            Resources.addSecureSearchPath(os.path.join(self._app_install_dir, "share", "cura", "resources"))
+            Resources.addSecureSearchPath(os.path.join(app_root, "Resources", "share", "cura", "resources"))
```

Comment:
This one is duplicated in the `if` and in the `else`, so it could be outside (or removed somewhere)

Reply:
oh yea, you're right. Let me change this

Reply:
https://github.com/Ultimaker/Cura/pull/18188/commits/d8cc99bf0deb951d32059e47b7690a0c76babfcb